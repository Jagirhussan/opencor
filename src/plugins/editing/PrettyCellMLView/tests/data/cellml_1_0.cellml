<?xml version="1.0" encoding="utf-8"?>
<model name="cellml_1_0" xmlns="http://www.cellml.org/cellml/1.0#" xmlns:cellml="http://www.cellml.org/cellml/1.0#" xmlns:cmeta="http://www.cellml.org/metadata/1.0#">
</model>
