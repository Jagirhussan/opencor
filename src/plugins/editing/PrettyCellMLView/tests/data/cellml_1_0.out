def model cellml_1_0 as
enddef;
